_api-reference/cluster-api/cluster-settings.md

```diff
@@ -81,6 +82,9 @@ The following request field parameters are compatible with the cluster API.
 | network.breaker.inflight_requests.limit | String | The limit for the in-flight requests breaker. Default is `100%` of the JVM heap. |
 | network.breaker.inflight_requests.overhead | Integer/Time unit | The constant that all in-flight request estimations are multiplied by to determine a final estimation. Default is `2`. |
 | script.max_compilations_rate | String | The limit for the number of unique dynamic scripts within a defined interval that are allowed to be compiled. Default is 150 every 5 minutes: `150/5m`. |
+| cluster.default.index.refresh_interval | Time unit | Sets the refresh interval what the `index.refresh_interval` setting is not provided when the index was created of what the existing `index.refresh_interval` setting is set to `null`. This setting is useful when you want to set a default refresh interval across all indexes in a cluster and also support the `searchIdle` setting. You can only set the interval as low as the `cluster.minimum.index.refresh_interval` setting. |
+| cluster.minimum.index.refresh_interval | Time unit | Sets the minimum refresh interval applicable for all indexes in the cluster. The `cluster.default.index.refresh_interval` setting should be higher than this setting's value. If, during index creation, the `index.refresh_interval` setting is lower than the minimum set, index creation fails. |
+| cluster.remote_store.translog.buffer_interval | Time unit | The default value of the translog buffer interval used when performing periodic translog updates. Use this setting only when the `index.remote_store.translog.buffer_interval` is not set inside the index's settings |
```

Comment:
`The default value of the translog buffer interval used when performing periodic translog updates.` -> The last section can be updated to `periodic updates to remote store.`

`Use this setting only when the `index.remote_store.translog.buffer_interval` is not set inside the index's settings` - This conveys a different meaning. Please refer https://github.com/opensearch-project/documentation-website/pull/4973#issuecomment-1711233429. `The setting is only effective if the index setting index.remote_store.translog.buffer_interval is not present with an index.`